### PR TITLE
[CDAP-14877] handle pagination and cursor in home page

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14877
Build: https://builds.cask.co/browse/CDAP-UDUT224

### CODE REVIEW

File: cdap-ui/app/cdap/components/PaginationStepper/index.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import * as React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import ChevronLeft from '@material-ui/icons/ChevronLeft';
+import ChevronRight from '@material-ui/icons/ChevronRight';
+
+interface IProps {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PaginationStepper/index.tsx]: nit: Can this be `IPaginationStepperProps`?

File: cdap-ui/app/cdap/components/PaginationStepper/index.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+      </IconButton>
+
+      <IconButton onClick={onNext} disabled={nextDisabled}>
+        <ChevronRight fontSize="small" />
+      </IconButton>
+    </div>
+  );
+};
+
+export default PagincationStepper;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/PaginationStepper/index.tsx]: Would it make sense to file a issue in material-ui for this component? Seems to be generic and simple enough.

Comment (elfenheart) [reply cdap-ui/app/cdap/components/PaginationStepper/index.tsx]: yeah, will file an issue

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Pull request merged**